confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeDescriptor.java

```diff
@@ -225,6 +225,12 @@ private void loadProps() {
               properties.getProperty(
                   "heartbeat_interval", String.valueOf(conf.getHeartbeatInterval()))));
 
+      conf.setUnknownDataNodeDetectInterval(
+          Long.parseLong(
+              properties.getProperty(
+                  "unknown_datanode_detect_interval",
+                  String.valueOf(conf.getHeartbeatInterval()))));
```

Comment:
Should we need to add another parameter? We should update iotdb-confignode.properties file if it's necessary to add this parameter.